src/templates/index.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>Welcome to forum123!</h1>
```

Comment:
This index page looks much better than just "Hello, world!", but let's make it in another task too keep diff small, see [comment](https://github.com/EugeneTurkin/forum123/pull/14/files#r1063904263)